packages/@okta/vuepress-theme-prose/assets/css/components/_button.scss

```diff
@@ -1,5 +1,5 @@
 .is-button-cerise-outline {
-  border-color: cv('cerise');
+  border-color: cv('header');
```

Comment:
A bit odd to use "header" color for buttons, any way we could harden these colour names?

Reply:
Reiterating this comment